src/hooks/useInputs.ts

```diff
@@ -27,13 +28,18 @@ export default function useInputs(): UseInputsReturnTypes {
     core.getInput(INPUT_KEYS.SEMANTIC_TITLE_REQUIRED, { required: true })
   );
 
+  const isSemanticPRTitleRequired = Boolean(
```

Comment:
```suggestion
  const isSemanticBranchNameRequired = Boolean(
```

burasını ve kodun devamını branch name kontrolü diye devam ettirsek daha iyi değil mi